apps/studio/data/platform/incident-status-utils.ts

```diff
@@ -0,0 +1,98 @@
+import dayjs from 'dayjs'
+
+import type { IncidentInfo } from './incident-status-query'
+
+/**
+ * Status priority order: higher priority = more urgent
+ * Used to determine which incident's status should be displayed
+ */
+const STATUS_PRIORITY: Record<string, number> = {
+  investigating: 4,
+  identified: 3,
+  monitoring: 2,
+  resolved: 1,
+}
```

Comment:
Please confirm this is the correct order of priority and full list, with `4` being the highest priority.